src/flow/src/adapter.rs

```diff
@@ -281,7 +282,7 @@ impl FlownodeManager {
                 let schema = meta.schema.column_schemas;
                 let is_auto_create = schema
                     .last()
-                    .map(|s| s.name == "__ts_placeholder")
+                    .map(|s| s.name == GREPTIME_TIMESTAMP)
```

Comment:
Will it introduce a compibility issue?

Reply:
> Will it introduce a compibility issue?

Indeed, and may cause other issues(like when user manually create a table with last col of name `GREPTIME_TIMESTAMP`, I should still instead use a column name that user usually will not  use like `__ts_placeholder`?

Reply:
> > Will it introduce a compibility issue?
> 
> Indeed, and may cause other issues(like when user manually create a table with last col of name `GREPTIME_TIMESTAMP`, I should still instead use a column name that user usually will not use like `__ts_placeholder`?

now change back to prevent breaking change, will change in 0.9 when flow can correctly auto deduce table schema